data/campaigns/World_Conquest/era/factions/Random.cfg

```diff
@@ -7,6 +7,6 @@
         image= units/random-dice.png
         random_faction=yes
         ## The empire is not complteley blaanced to exclude it for now from random.
-        except=The Empire
+        # except=The Empire
```

Comment:
Is it useful to keep this line and the comment above?

Reply:
> Is it useful to keep this line and the comment above?

The comments from the previous maintainer suggests that this still needs tweaking in terms of unit types, so I kept that commented out if the need to disable again arises.
(also, kind of practice to comment out unused code for future reference).

Reply:
The comment is wrong though, since it's not actually excluded anymore. I think it would make more sense to remove both these lines, and if you want to retain a note about the empire not being balanced, put it wherever the empire is defined.

Reply:
Okay, I have made a follow up commit to resolve this and the other responses